### ISSUE TITLE
[Feature]: Raw Zinc

### ISSUE THREAD

**Issue by FoggyWizard** [TheosCreation/CreateCompression#10]
### Feature Details

It would be useful for raw zinc to get compressed like the vanilla raw ore blocks can. Especially with mods like Create: Ironworks out, which require raw ore for making brass.

**Comment by TheosCreation** [comment]
Yes good idea and I'll have a look at ironworks too for the raw brass
